<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGitBranch</class>
 <widget class="QDialog" name="CreateGitBranch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>New branch name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="branchNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="gitLogMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="createBranchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>C&amp;reate</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createAndCheckoutButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Create and checkout</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>CreateGitBranch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
